src/ncp-spinel/SpinelNCPInstance.cpp

```diff
@@ -2396,18 +2396,20 @@ SpinelNCPInstance::handle_ncp_spinel_value_inserted(spinel_prop_key_t key, const
 	} else if (key == SPINEL_PROP_THREAD_ON_MESH_NETS) {
 		struct in6_addr *prefix = NULL;
 		uint8_t prefix_len = 0;
-		bool stable;
+		bool stable = false;
 		uint8_t flags = 0;
-		bool is_local;
+		bool is_local = false;
 
 		spinel_datatype_unpack(value_data_ptr, value_data_len, "6CbCb",
```

Comment:
I'm not sure if we should also check the return value here, which ensures the value of `prefix_length`, `stable`, `flags`, and `is_local` is properly parsed.

Reply:
You bring up a good point. I can't remember which fields we consider to be optional for this property. In this case we are considering all but the first to be "optional".

This is likely a wider subject that needs some more careful thought, since there are a lot of other cases that this could apply to. We can track that with a separate issue (Will file shortly).